controllers/dockermachine_controller.go

```diff
@@ -49,20 +52,20 @@ type DockerMachineReconciler struct {
 // +kubebuilder:rbac:groups=cluster.x-k8s.io,resources=clusters;machines,verbs=get;list;watch
 
 // Reconcile handles DockerMachine events
-func (r *DockerMachineReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, reterr error) {
-	ctx := context.Background()
-	log := r.Log.WithValues("dockermachine", req.NamespacedName)
+func (r *DockerMachineReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, rerr error) {
+	ctx := context.TODO()
+	log := log.Log.WithName(machineControllerName).WithValues("dockerMachine", req.NamespacedName)
```

Comment:
I've been using `skewer-style` (css-style) for the logging keys. We should pick a style and stick with it. Do you mind changing this to skewer-case for consistency and then opening an issue so we can evaluate what people want, probably at the CAPI meeting?

Reply:
+1 to open an issue. ATM I will align to skewer style